controllers/clustergroupupgrade_controller.go

```diff
@@ -174,13 +174,21 @@ func (r *ClusterGroupUpgradeReconciler) Reconcile(ctx context.Context, req ctrl.
 				allManagedPoliciesExist, managedPoliciesMissing, managedPoliciesPresent, err :=
 					r.doManagedPoliciesExist(ctx, clusterGroupUpgrade, true)
 				if err != nil {
+					if _, ok := err.(utils.PolicyStatusUnknownError); ok {
```

Comment:
Not sure if we need to have a specific requeue after value for this case, as we can just treat it as non-compliant. Why not just do that in getClusterComplianceWithPolicy and getClustersNonCompliantWithPolicy. We already do that for the case where the map is there but the cluster entry doesn't exist.